It was applied, then reverted (by mistake?) in 4d2c55348bc014.

It's been resubmitted here:
https://lists.gnu.org/archive/html/jami/2024-01/msg00005.html

The remaining hunk was reverted by 22be4be8.
Submitted:
https://git.jami.net/savoirfairelinux/jami-client-qt/-/issues/2112

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 29bfde9a..3b4c7a92 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -237,6 +237,6 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/extras/build/cmake/modules)
 if (NOT JAMICORE_AS_SUBDIR)
   find_package(LibJami REQUIRED)
   if(LIBJAMI_FOUND)
-    include_directories(${LIBJAMI_INCLUDE_DIRS})
+    include_directories(${LIBJAMI_INCLUDE_DIR})
   endif()
 endif()
